pyaedt/modeler/Model2D.py

```diff
@@ -177,3 +177,57 @@ def radial_split_2D(self, radius, name):
 
         self.subtract(name, name + "1")
         return True
+
+    @pyaedt_function_handler()
+    def objects_in_bounding_box(self, bounding_box, check_lines=True, check_sheets=True):
+        """Given a 2D bounding box checks if sheets and lines are inside it.
+
+        Parameters
+        ----------
+        bounding_box : list.
+            List of either 4 or 6 coordinates of bounding box.
+        check_lines : bool, optional.
+            Check line objects.
+        check_sheets : bool, optional.
+            Check sheet objects.
+
+        Returns
+        -------
+        list of :class:`pyaedt.modeler.Object3d`
+        """
+
+        if len(bounding_box) != 4:
+            raise ValueError("Bounding box list must have dimension 4.")
```

Comment:
Should this be written as is or rather as "Bounding box list must have four dimensions."

Reply:
I meant to say that bounding box list must be a list of 4 elements.
Any suggestion is welcome!